tripal_chado/src/Plugin/Field/FieldType/ChadoStockTypeDefault.php

```diff
@@ -330,4 +330,28 @@ public function isCompatible(TripalEntityType $entity_type) : bool {
     return $compatible;
   }
 
+  /**
+   * {@inheritDoc}
+   * @see \Drupal\tripal\TripalField\Interfaces\TripalFieldItemInterface::discover()
+   */
+  public static function discover(TripalEntityType $bundle, string $field_id, array $field_types,
+      array $field_instances, array $options = []): array {
+
+    // Specific settings for this field
+    // There are four stock types in core Tripal, use the most generic one
+    $options += [
+      'id' => self::$id,
+      'table' => self::$object_table,
+      'label' => 'Stock',
```

Comment:
I really don't agree with calling this stock as it's too chado-specific and while it's intuitive to admin it would not be to content editors / curators. What about instead using the category name as we do in the add content list and structure lists?

```suggestion
      'label' => 'Germplasm',
```

<img width="1268" alt="Screenshot 2025-01-24 at 9 57 11 AM" src="https://github.com/user-attachments/assets/e3480481-308e-44f9-9a6b-d61ddf9e5f77" />
<img width="1302" alt="Screenshot 2025-01-24 at 9 57 25 AM" src="https://github.com/user-attachments/assets/4d6dd69d-9050-4e1f-acbb-d5a3d80330ee" />

Reply:
Sounds good